.gitignore

```diff
@@ -23,3 +23,4 @@ testbin/*
 *.swp
 *.swo
 *~
+.vscode/**
```

Comment:
Please remove this commit from the PR. It is unrelated to metrics

Reply:
Done